app/src/main/java/com/andrew/tasky/domain/AgendaItems.kt

```diff
@@ -44,28 +44,31 @@ object AgendaItems {
         )
     )
 
-    //Returns a sorted list of agendaItems by start time and only those of dateSelected
-    fun sortByDateSelected(dateSelected: LocalDate): MutableList<AgendaItem>{
+    // Returns a sorted list of agendaItems by start time and only those of dateSelected
+    fun sortByDateSelected(dateSelected: LocalDate): MutableList<AgendaItem> {
         val agendaItemListSorted = mutableListOf<AgendaItem>()
-        agendaItemList.sortedBy{ agendaItem -> agendaItem.startDateAndTime }
-            .forEach{if (it.startDateAndTime.toLocalDate() == dateSelected){
-                agendaItemListSorted.add(it)}}
+        agendaItemList.sortedBy { agendaItem -> agendaItem.startDateAndTime }
+            .forEach {
+                if (it.startDateAndTime.toLocalDate() == dateSelected) {
+                    agendaItemListSorted.add(it)
+                }
+            }
         return agendaItemListSorted
```

Comment:
Better directly avoid using mutable lists because race conditions can easily happen:
```
return agendaItemList.sortedBy { agendaItem ->
    agendaItem.startDateAndTime
}.filter {
    it.startDateAndTime.toLocalDate() == dateSelected
}
```